### ISSUE TITLE
[A11y] Desktop/Mobile Collection Landing Page Collection Name

### ISSUE THREAD

**Issue by azle** [OregonDigital/OD2#1748]
### Operating System

MacOS

### Browser

Chrome

### Affected page

Collection Show

### Admin user?

- [ ] Yes

### Issue Description

Header 1 Collection Title "Gerald W. Williams Collection" is missing aria-label for collection name.

**Comment by CGillen** [comment]
@azle Can you expand on this a little? The `<h1>` should be a label in its self and should need an aria-label. Am I missing something about this header in particular?

**Comment by azle** [comment]
No, your right h1 doesn't need an aria label. I wrote this awhile back. I recall that I had an issue with the title but I just revalidated and not getting the aria label message.
